ops/log.py

```diff
@@ -26,7 +26,21 @@ def emit(self, record):
         self.model_backend.juju_log(record.levelname, self.format(record))
 
 
-def setup_root_logging(model_backend):
+def setup_root_logging(model_backend, *, debug_stream=None):
+    """Setup python logging to forward messages to juju-log.
+
+    :param model_backend: a ModelBackend to use for juju_log.
+    :param debug_stream: (optional) If set, debug messages will also be sent to this stream.
```

Comment:
The "(optional)" and "If set," are redundant.

Is "(optional)" a standard/agreed way to specify that?  If so, sounds good, otherwise we should either try to agree on that or drop it and allow the sentence to describe it.

Reply:
Since you raise the question, Sphinx seems to say that you should put `, optional` into the type:
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html
And ', defaults to' into the string description.